package.json

```diff
@@ -108,6 +108,7 @@
     "jsonpath-plus": "^0.16.0",
     "jsonwebtoken": "^8.2.1",
     "lodash": "^4.17.4",
+    "trim-newlines": "^2.0.0",
```

Comment:
This seems small enough that we could just implement it ourselves? No need to add this dependency when we could do something like this (untested)
```js
const trimNewLines = str => str.replace(/^\r?\n|\r?\n$/g, '');
```
Don't want to have something like [the left-pad debacle](https://www.theregister.co.uk/2016/03/23/npm_left_pad_chaos/) happen.

Reply:
This is the library code itself:
```node
x => x.replace(/^[\r\n]+/, '').replace(/[\r\n]+$/, '');
```